standalone-metastore/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreSchemaInfo.java

```diff
@@ -137,6 +138,17 @@ public String generateInitFileName(String toVersion) throws HiveMetaException {
     return initScriptName;
   }
 
+  @Override
+  public String getCreateUserScript() throws HiveMetaException {
```

Comment:
Do we need to change getMetaStoreScriptDir() method implementation as well since we changed the scripts location?

Reply:
I don't think so, as getMetaStoreScriptDir find the location of the scripts in the distribution (not the source code) and I made sure to leave the location of the scripts the same in the distribution.  hiveHome is a poor name this variable since it is really related to where the metastore is, not where hive is.  I'll switch the name to metastoreHome.